Uniswap/interface
# fix: Revert to previous Nav menu

Reverts to the Nav UI from prior to https://github.com/Uniswap/interface/pull/6216

Before (app.uniswap.org):
<img width="497" alt="Screen Shot 2023-03-27 at 9 37 17 AM" src="https://user-images.githubusercontent.com/4899429/227954630-23db7f29-db43-474b-8423-5b9a1e5af666.png">
Note, the tax services 

After:
<img width="440" alt="Screen Shot 2023-03-27 at 9 37 12 AM" src="https://user-images.githubusercontent.com/4899429/227954700-a1632903-d622-4530-877c-fc2ef6a9fab4.png">

Need to check:
- Are we keeping the tax services row?
- Do we need to show Pools in menu when on truncated view?

Test:
- Open menu on preview
- Click on the various icons to make sure they work
